### PR TITLE
3 detail card nft images

### PR DESCRIPTION
**Pull request by philipperker**
In this user story I have added a detail card and the navbar.

### CODE REVIEW

File: client/src/components/CardSlide/CardSlide.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
 import { useState, useEffect } from 'react';
 import styled from 'styled-components';
+import PopUpDetailsNFT from '../PopUpDetailsNFT/PopUpDetailsNFT';
 
 export default function CardSlide() {
   const [data, setData] = useState([]);
   const [error, setError] = useState(null);
 
+  //--- Test PopUP Details NFT
+  const [isOPen, setIsOpen] = useState(false);
```
Comment (PulexIrritans) [inline client/src/components/CardSlide/CardSlide.js:10]: Do you need this comment, still?

File: client/src/components/PopUpDetailsNFT/PopUpDetailsNFT.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function PopUpDetailsNFT({
+  open,
+  nftTokenId,
+  nftImageUrl,
+  nftSmartContact,
+  onClose,
+  createdBy,
```
Comment (PulexIrritans) [inline client/src/components/PopUpDetailsNFT/PopUpDetailsNFT.js:10]: If you use an object you could destructure it here.

File: client/src/components/PopUpDetailsNFT/PopUpDetailsNFT.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <Content>Token-ID: {nftTokenId}</Content>
+      <Content>Smart Contact: </Content>
+      <Content>{nftSmartContact.slice(0, 20) + '....'}</Content>
+    </PopUp>
+  );
+}
+
+const PopUp = styled.div`
+  /* display: flex;
+  flex-direction: column; */
```
Comment (PulexIrritans) [inline client/src/components/PopUpDetailsNFT/PopUpDetailsNFT.js:30]: What is this: /*... */ ?

File: client/src/pages/NFTview.js [hunk 4]
```diff
@@ -12,15 +13,14 @@ export default function NFTview() {
...
+      <Navigation />
     </NFTviewMain>
   );
 }
 
 const NFTviewMain = styled.main`
+  z-index: 2;
   display: grid;
   grid-template-columns: 1fr auto 1fr;
   grid-template-rows: auto; ;
```
Comment (PulexIrritans) [inline client/src/pages/NFTview.js:26]: One lost semicolon.

**Review by PulexIrritans — COMMENTED** [review]
Hi Philipp,
Nice job. I especially liked how you fetch your data with the async in the useEffect. I have learned sth here. Also, I would like to know why you use the {children} in the button. 

Now I'm really eager to see your current project state. Keep it up. We'll all make it. :-)

File: client/src/components/CardSlide/CardSlide.js [hunk 5]
```diff
@@ -1,10 +1,18 @@
...
+  const [tokenId, setNFTtokenID] = useState({});
+  const [imageURL, setImageURL] = useState({});
+  const [smartContact, setSmartContact] = useState({});
+  const [createdBy, setCreatedBy] = useState({});
+
```
[the remark this answers is not in the document]
Comment (philipperker) [reply client/src/components/CardSlide/CardSlide.js:15]: Thank you Anjte for your feedback:) You're right, that makes my code cleaner. I will change it in the next user story.

File: client/src/components/CardSlide/CardSlide.js [hunk 6]
```diff
@@ -1,10 +1,18 @@
 import { useState, useEffect } from 'react';
 import styled from 'styled-components';
+import PopUpDetailsNFT from '../PopUpDetailsNFT/PopUpDetailsNFT';
 
 export default function CardSlide() {
   const [data, setData] = useState([]);
   const [error, setError] = useState(null);
 
+  //--- Test PopUP Details NFT
```
[the remark this answers is not in the document]
Comment (philipperker) [reply client/src/components/CardSlide/CardSlide.js:9]: No, I will delete the comment

File: client/src/components/Navigation/Navigation.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    <Navbar>
+      <Button children={'home'} />
+    </Navbar>
+  );
+}
+
+const Navbar = styled.div`
+  z-index: 1;
+  display: flex;
+  flex-direction: row;
```
[the remark this answers is not in the document]
Comment (philipperker) [reply client/src/components/Navigation/Navigation.js:16]: You're right. I don't need it for the one button. But for the next button;)

File: client/src/components/PopUpDetailsNFT/PopUpDetailsNFT.js (hunk 3 above)
Comment (philipperker) [reply client/src/components/PopUpDetailsNFT/PopUpDetailsNFT.js:30]: The content of the comment:)?

**Pull request merged**